machines/vcItem.ts

```diff
@@ -1003,16 +1007,26 @@ export const vcItemMachine =
           { to: (context) => context.serviceRefs.store }
         ),
 
-        loadMyVcs: send(StoreEvents.GET(MY_VCS_STORE_KEY), {
-          to: (context) => context.serviceRefs.store,
-        }),
-
```

Comment:
why this code is removed?

Reply:
We are not using this any where, we can removed it.